cluster-sync/external/README.md

```diff
@@ -7,30 +11,26 @@ such that:
 2. We generate the manifests with the provided DOCKER_PREFIX (uses default port)
 3. Uses your configured `kubectl` and deploys your build to the existing cluster
 
-# Building images and pushing to your registry
-
-```bash
-export DOCKER_PREFIX=index.docker.io/barney_rubble
-export DOCKER_TAG=latest # defaults to `latest`
-export KUBEVIRT_PROVIDER=external
-```
-
-`make docker push`
-
 # Build and push images, create manifests and deploy CDI
 
-We use the same workflow as the ephemeral dev environment, but skip `make cluster-up`:
+We use the same workflow as the ephemeral dev environment, but skip `make cluster-up`.  
+A suggested flow:
 
 ```bash
-export DOCKER_PREFIX=index.docker.io/barney_rubble
-export DOCKER_TAG=latest # defaults to `latest`
 export KUBEVIRT_PROVIDER=external
+export PULL_POLICY=Always
+export CDI_NAMESPACE=cdi-rand-ns-name
```

Comment:
instead of cdi-rand-ns-name, I would say something like namespace cdi is installed into. Defaults to 'cdi'

Reply:
I think going with random is a good practice for external clusters, completes the package of testing CDI with ensuring we are name agnostic

Reply:
This is the documentation on how to work with an external cluster, the namespace will be pretty set in that scenario. We randomize in our CI anyway to catch hardcoding.

Reply:
I just think its a good idea to go random here instead of default..

Reply:
The ns will not be set in that scenario, I am working on a clean openshift cluster and dumping CDI into a fresh ns